Products/CMFPlone/controlpanel/tests/test_controlpanel_browser_usergroups.py

```diff
@@ -1,13 +1,12 @@
-from plone.app.testing import SITE_OWNER_NAME
-from plone.app.testing import SITE_OWNER_PASSWORD
+import unittest
+
+import transaction
+from plone.app.testing import SITE_OWNER_NAME, SITE_OWNER_PASSWORD
 from plone.testing.zope import Browser
 from Products.CMFCore.utils import getToolByName
 from Products.CMFPlone.testing import PRODUCTS_CMFPLONE_FUNCTIONAL_TESTING
 from Products.CMFPlone.utils import normalizeString
 
-import transaction
-import unittest
```

Comment:
The imports should have remained at this place.  Running either `isort` or `pre-commit run -a` should have fixed that.
But that is a detail, and it is not fully enforced yet in this repo I think.

Reply:
Actually, let me run that on your branch.  Done and pushed.